text_mining.py

```diff
@@ -0,0 +1,217 @@
+"""Mines text from Project Gutenburg. Performs word frequency and sentiment analysis
+on text and generates a word cloud from most common words.
+
+@author: Vivien Chen
+
+"""
+
+import requests
+import string
+import math
+from pickle import dump, load
+from os.path import exists
+from stop_words import get_stop_words       # pip install stop_words
+from vaderSentiment.vaderSentiment import SentimentIntensityAnalyzer
+from wordcloud import WordCloud             # pip install wordcloud
+
+
+def get_cache(url, file_name):
+    """Writes text from url into file_name and returns text if file_name does not exist.
+    Otherwise, simply returns text from file_name. Assumes that text from file_name
+    matches text from url if file_name exists.
+
+    Args:
+        url: url you want to read text from
+        file_name: name of file you want to write to/read from
+
+    Returns:
+        text from url
+    """
+    if exists(file_name) == False:
```

Comment:
Small style thing - rather than checking if a boolean is equal to false, we can just do "if not exists(filename) :" or "!exists".

Reply:
Side note, I like the structure of reading a file unless the file doesn't exist, then grabbing from the URL instead. It makes the program nice and portable!